charts/values.yaml

```diff
@@ -0,0 +1,26 @@
+controllerManager:
+  kubeRbacProxy:
+    image:
+      repository: gcr.io/kubebuilder/kube-rbac-proxy
+      tag: v0.13.1
+    resources:
+      limits:
+        cpu: 500m
+        memory: 128Mi
+      requests:
+        cpu: 5m
+        memory: 64Mi
+  manager:
+    image:
+      repository: registry.woqutech.com/xiewei/opengemini-operator
+      tag: 814de3b
```

Comment:
we can now use `ghcr.io/opengemini/opengemini-operator:latest` as default operator image